src/core/server/http/http_config.ts

```diff
@@ -19,6 +19,7 @@
 
 import { ByteSizeValue, schema, TypeOf } from '@kbn/config-schema';
 import { Env } from '../config';
+import { DEFAULT_CSP_RULES, DEFAULT_CSP_STRICT, DEFAULT_CSP_WARN_LEGACY_BROWSERS } from '../csp';
```

Comment:
why default values are under separate domain?

Reply:
They are defined under a separate domain in legacy. Would you like me to move these to HTTP?

Reply:
This is how it was laid out in legacy, but we can solve this in the CSP changes like you mentioned later.

Reply:
FYI: We did this in the legacy world so that we could add the security team as CODEOWNERS to the CSP defaults. I think that's something we'd want to keep doing in the new platform as well.

Reply:
we can make them owners for a nested folder. don't we? 
```
# Order is important; the last matching pattern takes the most
# precedence.
```
also we can declare several codeowners
```
# These owners will be the default owners for everything in
# the repo. Unless a later match takes precedence,
# @global-owner1 and @global-owner2 will be requested for
# review when someone opens a pull request.
```
https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

Reply:
As far as I know, yes we can do that. The important bit is that the default values be in a file that is not heavily coupled to http service code, but I see no reason why it can't exist inside the http module.